timer.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Timer</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body class="background">
+    <div class="timers" id="timers">
+     <h1>Timer</h1>
+      <p id="count">0</p>
+        <button id="startBtn">Start</button>
```

Comment:
Kindly use snake-case for ID.

Refer to this guidelines:
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5